### PR TITLE
Login and Registration form

### PR DESCRIPTION
**Pull request by Reverrd**
some few merging issues still remain tho

### CODE REVIEW

File: src/components/authentication/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+// import { useState } from 'react';
+import { useEffect, useState, FormEvent } from 'react';
+import './auth.scss';
+// import Register from './Register';
+import AppleIcon from '@mui/icons-material/Apple'
+interface RegisterProps{
```
Comment (DaScOrPiO) [inline src/components/authentication/Login.tsx:6]: Move all type definitions to utils/types/todo and import in components

File: src/components/authentication/index.tsx [hunk 2]
```diff
@@ -1,3 +1,23 @@
+import { useState } from 'react'
+import Login from './Login'
+import Register from './Register'
+
+
 export default function Index() {
-  return <div>index</div>;
+  const [isLogin, setIsLogin] = useState(true)
```
Comment (DaScOrPiO) [inline src/components/authentication/index.tsx:7]: Declare appropriate types for all state variables
for instance: `const [isLogin, setIsLogin] = useState<boolean>(true)`

File: src/components/authentication/index.tsx [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+   const [isRegister, setIsRegister] = useState(false)
```
Comment (DaScOrPiO) [inline src/components/authentication/index.tsx:8]: same here

File: src/components/authentication/Register.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import './auth.scss';
+import AppleIcon from '@mui/icons-material/Apple'
+interface LoginProps{
```
Comment (DaScOrPiO) [inline src/components/authentication/Register.tsx:3]: same here move to utils/types/todo.ts

File: src/components/authentication/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
+// import { useState } from 'react';
+import { useEffect, useState, FormEvent } from 'react';
+import './auth.scss';
```
Comment (DaScOrPiO) [inline src/components/authentication/Login.tsx:3]: Create a new folder named authentication in styles directory, move all stylesheets there and import for use where necessary.

**Review by DaScOrPiO — CHANGES_REQUESTED** [review]
Nice work! Please make these few changes.

File: src/components/authentication/index.tsx (hunk 2 above)
Comment (Reverrd) [reply src/components/authentication/index.tsx]: resolved

File: src/components/authentication/Register.tsx (hunk 4 above)
Comment (Reverrd) [reply src/components/authentication/Register.tsx:5]: can you elaborate more on this?

Comment (Reverrd) [reply src/components/authentication/Register.tsx]: done

**Review by Reverrd — COMMENTED** [review]
viewed all

**Review by Reverrd — COMMENTED** [review]
review the pull request

**Review by Reverrd — COMMENTED** [review]
view changes

**Review by Reverrd — COMMENTED** [review]
reviewed changes

**Pull request merged**